### ISSUE TITLE
IterableOfLongs.java:35-39: Introduce IteratorOfLongs and...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/cactoos#838]
The puzzle `802-6284bb55` from #802 has to be resolved:

https://github.com/yegor256/cactoos/blob/9346b81018b7e8238f796ea6b59c833d287ba836/src/main/java/org/cactoos/iterable/IterableOfLongs.java#L35-L39

The puzzle was created by Krzysztof Krason on 08-May-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C63314D6Z) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#838](https://github.com/yegor256/cactoos/issues/838) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#838](https://github.com/yegor256/cactoos/issues/838) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/1281348710); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by victornoel** [comment]
@0crat waiting on #841

**Comment by victornoel** [comment]
@0crat waiting for #854 to be merged

**Comment by 0crat** [comment]
> @0crat waiting for #854 to be merged ([here](https://github.com/yegor256/cactoos/issues/838#issuecomment-388034505))

@victornoel Job [#838](https://github.com/yegor256/cactoos/issues/838) is already on hold

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by elenavolokhova** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)
